web/template/layout/edit-panel.gotmpl

```diff
@@ -16,48 +16,36 @@
           <h3 class="govuk-heading-s govuk-!-margin-0">Edit tasks</h3>
         </div>
       </header>
-          <div class="govuk-width-container">
-          <form action="{{ prefix "/" }}" method="POST" id="reassign-task-form" class="ignore-pagination-bottom">
-            <h4 class="govuk-heading-s"><span class="count-checked-checkboxes js-mt-task-count">0</span> tasks selected</h4>
-            <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
-              <label class="govuk-label" for="sort">
-                Assign to a team
-              </label>
-              <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"  class="js-xsrfToken"/>
-              <label for="assignTeam"></label> <select class="govuk-select option-value-select js-assign-team-select" data-module="assign-team-select" id="assignTeam" name="assignTeam">
-                <option value="0" selected>Select a team</option>
-                  {{ range .TeamSelection }}
-                  <option class="option-value" value="{{ .Id }}">{{ .Name }}</option>
-                  {{ end }}
-              </select>
-            </div>
+      <div class="govuk-width-container">
+        <div id="reassign-task-form" class="ignore-pagination-bottom">
+        <h4 class="govuk-heading-s"><span class="count-checked-checkboxes js-mt-task-count">0</span> tasks selected</h4>
+        <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
+          <label class="govuk-label" for="sort">
+            Assign to a team
+          </label>
+          <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"  class="js-xsrfToken"/>
+          <label for="assignTeam"></label> <select class="govuk-select option-value-select js-assign-team-select" data-module="assign-team-select" id="assignTeam" name="assignTeam">
```

Comment:
From memory this label exists because they didn't want any data showing in the list before the drop down was populated with data from the fetch request. Without this label the drop down would always have shown a case managers name at the top.